.automation/build.py

```diff
@@ -555,9 +559,6 @@ def generate_linter_dockerfiles():
         )
         # Browse descriptor linters
         for linter in descriptor_linters:
-            # Do not build standalone linter if it does not manage SARIF
-            if linter.can_output_sarif is False:
```

Comment:
@nvuillam I can wait to merge the branch when you have ready the `upload-docker.sh` refactoring to the native actions you said you were going to put in shortly.

That way we don't have to do any temporary hacks.

I'll also tell you that during my testing I haven't encountered the API limit as many times as you would think either.

Reply:
I don't know when my refacto of docker build will be started and ready, so i think it's better to merge your PR asap ... bit to do that we need less jobs ^^
Or at least add some condition so they are run only if the commit contains some words ^^

Reply:
@nvuillam I think this no longer applies since we have moved to docker-build-push-action + authenticate API calls from Github + phive and I can leave it without that condition.

The last build I ran didn't fail anything at all, it worked all first time.